cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-github-enterprise/results.txt

```diff
@@ -59,15 +58,38 @@ First-Party Blocking Rules Fired:
 === end of stdout - plain
 
 === stderr - plain
-Scan environment:
-  versions    - semgrep <MASKED> on python <MASKED>
-  environment - running in environment github-actions, triggering event is push
-  server      - https://semgrep.dev
-
+                                                                                
+                                                                                
+╭────────────────╮                                                              
+│ Debugging Info │                                                              
+╰────────────────╯                                                              
+                                                                                
+  SCAN ENVIRONMENT                                                              
+  versions    - semgrep <MASKED> on python <MASKED>                                  
+  environment - running in environment github-actions, triggering event is      
+                push                                                            
+  server      - https://semgrep.dev                                             
+                                                                                
+                                                                                
+╭─────────────╮                                                                 
+│ Scan Status │                                                                 
+╰─────────────╯                                                                 
+                                                                                
 Fetching configuration from semgrep.dev
 Authenticated as org_name
 Scanning 1 file with 4 python rules.
-
+                                                                                
+                                                                                
+╭─────────╮                                                                     
+│ Results │                                                                     
+╰─────────╯                                                                     
```

Comment:
Should we print results title when there are no results?

Reply:
yeah good point, probably not

Reply:
I tested locally and it's not printing this heading if there are no results